README.md

```diff
@@ -69,6 +70,15 @@ Access Generative AI models via SDKs of popular cloud service providers.
 
 - [Pinecone.jl](https://github.com/tullytim/Pinecone.jl) - SDK for [Pinecone.io](https://www.pinecone.io/) vector database.
 
+
+### General purpose DBMS with vector index support
+| Name                  | Julia client                | Usage examples                |
+|:---------------------:|:---------------------------:|:-----------------------------:|
+| [Elasticsearch](https://www.elastic.co/elasticsearch) | [ElasticsearchClient.jl](https://github.com/OpenSesame/ElasticsearchClient.jl) | [GptSeachPlugin with Elasticsearch](https://github.com/OpenSesame/GptSearchPlugin/tree/main/ext/ElasticsearchClientExt)  |
+| [OpenSearch](https://opensearch.org/) | [ElasticsearchClient.jl](https://github.com/OpenSesame/ElasticsearchClient.jl) | [GptSeachPlugin with Opensearch](https://github.com/OpenSesame/GptSearchPlugin/tree/main/ext/OpenSearchExt)  |
```

Comment:
OpenSearch is a fork of ElasticSearch. The library uses the same transport layer. But there are differences in the query languages for these products. Therefore, the library is the same, but the examples are different.

Reply:
Also, since I created the commit, there are some changes in the namespaces. It is moved out from the OpenSesame org account to private accounts:
https://github.com/LarsWl/ElasticsearchClient.jl
https://github.com/rssdev10/GptSearchPlugin

Reply:
Will you update the links, please?

Reply:
done